plugin/src/test/sc/plugin2021/GameTest.kt

```diff
@@ -32,4 +34,24 @@ class GameTest: StringSpec({
             }
         }
     }
+    "A game of skips eventually ends in a draw" {
+        val game = Game()
+        val state = game.gameState
+        Pair(game.onPlayerJoined(), game.onPlayerJoined())
+        game.start()
+
+        for (s in 0 until 4)
+            game.onAction(state.currentPlayer, GameRuleLogic.streamPossibleMoves(state).first())
+        
+        shouldNotThrowAny {
+            while (!game.isGameOver())
+                game.onAction(state.currentPlayer, SkipMove(state.currentColor))
+        }
+        shouldThrow<java.lang.IndexOutOfBoundsException> {
+            game.onAction(state.currentPlayer, SkipMove(state.currentColor))
+        }
+    
+        game.playerScores shouldContainExactly List(2)
+        {PlayerScore(ScoreCause.REGULAR, "", Constants.DRAW_SCORE, 10)}
```

Comment:
missing continuation indent really confused me here

Reply:
this is indented properly now